### ISSUE TITLE
Unsupervised Pre-training Results

### ISSUE THREAD

**Issue by YilmazKadir** [OpenGVLab/PonderV2#23]
Thank you for sharing your code! From my understanding, you utilize the semantic labels from multiple datasets by leveraging their corresponding CLIP features, similar to the approach in the Point Prompt Training paper. I was wondering if you might have any results for cases where you didn’t use semantic labels but instead relied solely on rendering as a pretraining task?

**Comment by HaoyiZhu** [comment]
Hi, thanks for your question! We haven't done that yet since we build our work upon the existing codebase (pointcept). However, you could definitely try it by, for example, one head per dataset. I believe the results will not have a significant difference.
